flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/sql/ml/SqlMLPredictTableFunction.java

```diff
@@ -134,100 +123,13 @@ public SqlOperandCountRange getOperandCountRange() {
 
         @Override
         public boolean isOptional(int i) {
-            return i > getOperandCountRange().getMin() && i <= getOperandCountRange().getMax();
+            return i >= getOperandCountRange().getMin() && i <= getOperandCountRange().getMax();
```

Comment:
i > getOperandCountRange().getMin() && i <= getOperandCountRange().getMax()

Reply:
I'm confused. If `i` is operand index, it should be `>=` min count? But it's `>` in `SqlWindowTableFunction`.

Reply:
I think you are right. Can you add a seperate commit to fix the problem in SqlWindowTableFunction?

Reply:
Calctie uses 0 to indicate the first operand here. It's better we can add a case to prevent other break this.